_data/master/navbars/networking.yml

```diff
@@ -56,3 +56,6 @@ toc:
       path: /networking/design/l2-interconnect-fabric
     - title: Calico over IP fabrics
       path: /networking/design/l3-interconnect-fabric
+- title: Accelerating Istio network performance
+  path: /networking/sidecar-acceleration
+  
```

Comment:
Could you remove the extra newlines from the end of the file? thanks!

Reply:
Removed. This is strange because first time Semaphore failed saying I did not have extra lines at the end of this file, so I added extra line. Then it was successful.